packages/core/src/codewhispererChat/view/connector/connector.ts

```diff
@@ -132,6 +134,68 @@ export class OpenSettingsMessage extends UiMessage {
     override type = 'openSettingsMessage'
 }
 
+export class ContextCommandData extends UiMessage {
+    readonly data: MynahUIDataModel['contextCommands']
+    override type = 'contextCommandData'
+    constructor(data: MynahUIDataModel['contextCommands']) {
+        super('tab-1')
```

Comment:
is this meant to only be `tab-1`?

Reply:
This tab-1 is a placeholder. We do not need tabId for this object but the current design pattern forces us to have a tabId in the UiMessage

Reply:
We can use an empty string instead.